examples/QuickJournal/ViewModels/JournalEntriesViewModel.cs

```diff
@@ -0,0 +1,80 @@
+﻿using CommunityToolkit.Maui.Alerts;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using CommunityToolkit.Mvvm.Messaging;
+using QuickJournal.Messages;
+using QuickJournal.Models;
+using Realms;
+
+namespace QuickJournal.ViewModels
+{
+    public partial class JournalEntriesViewModel : ObservableObject
+    {
+        private readonly Realm realm;
+
+        [ObservableProperty]
```

Comment:
Does this need to be observable property?

Reply:
In this case we don't, but I think it's a good pattern nevertheless. If for example we want to use `Realm.GetInstanceAsync` and we want to do it in `OnAppearing`, then we need it.